scipy/signal/wavelets.py

```diff
@@ -308,14 +309,147 @@ def ricker(points, a):
     return total
 
 
-def cwt(data, wavelet, widths):
+def morlet2(M, s, w=5):
```

Comment:
I think this is needed for compatibility with `cwt` (Continuous Wavelet Transform).